hack/rpm-deps.sh

```diff
@@ -222,7 +223,8 @@ bazel run \
     //:bazeldnf -- rpmtree --public --arch=aarch64 --name handlerbase_aarch64 \
     --basesystem centos-stream-release \
     --repofile rpm/centos-repo.yaml \
-    $basesystem \
+    $centos_base \
+    $centos_extra \
```

Comment:
`$basesystem` is dropped in "rpm: Reorganize package lists", so this should have been changed to

```
$fedora_base \
$fedora_extra \
```

in that commit and then to

```
$centos_base \
$centos_extra \
```

in the commit "rpm: Switch to CentOS Stream 8" along with all other invocations of "bazeldnf rpmtree".

Reply:
Fixed.